docs/fundamentals/environments.rst

```diff
@@ -1,8 +1,127 @@
-.. include:: /stub-topic.txt
+Working with Multiple Environments
+==================================
 
-|stub-icon| Working with Multiple Environments
-==============================================
+By :ref:`Steve Smith <environments-author>` | Originally Published: 18 May 2015 
 
-.. include:: /stub-notice.txt
+ASP.NET 5 introduces improved support for controlling application behavior across multiple environments, such as development, staging, and production. Environment variables are used to indicate which environment the application is running in, allowing the app to be configured appropriately.
 
-.. _issue: https://github.com/aspnet/Docs/issues/70
\ No newline at end of file
+In this article:
+	- `Development, Staging, Production`_
+	- `Startup conventions`_
+	- `Environment variables`_
+
+`Browse or download samples on GitHub <https://github.com/aspnet/Docs/tree/master/docs/fundamentals/environments/sample>`_.
+
+Development, Staging, Production
+--------------------------------
+
+ASP.NET 5 references a particular `environment variable <https://github.com/aspnet/Home/wiki/Environment-Variables>`_, ``ASPNET_ENV``, to describe the environment the application is currently running in. This variable can be set to any value you like, but three values are used by convention: ``Development``, ``Staging``, and ``Production``. You will find these values used in the samples and templates provided with ASP.NET 5. Since ASP.NET 5 is open source, you can easily view all usages of these three environment names by `searching the source code on GitHub <https://github.com/search?utf8=%E2%9C%93&q=staging+development+production+user%3Aaspnet&type=Code&ref=searchresults>`_.
+
+The current environment setting can be detected programmatically from within ASP.NET 5. In addition, ASP.NET MVC 6 introduces an `Environment Tag Helper <http://docs.asp.net/en/latest/mvc/views/tag-helpers/index.html>`_ that allows MVC Views to include certain sections based on the current application environment.
+
+.. note:: Like all environment variables, these values are case-insensitive. Whether you set the variable to ``Development`` or ``development`` or ``DEVELOPMENT`` the results will be the same.
+
+Development
+^^^^^^^^^^^
+
+This should be the default environment used when developing an application. When using Visual Studio 2015, this setting is specified by default in the project's properties for IIS Express, as shown here:
```

Comment:
this setting is specified by default in the project's properties for IIS Express -> this setting can be specified in your project's debug profile